interactive-clap-derive/src/derives/interactive_clap/mod.rs

```diff
@@ -1,241 +1,59 @@
 extern crate proc_macro;
 
-use methods::cli_field_type;
 use proc_macro2::{Span, TokenStream};
 use proc_macro_error::abort_call_site;
 use quote::{quote, ToTokens};
 use syn;
 
-use crate::LONG_VEC_MUTLIPLE_OPT;
+#[cfg(test)]
```

Comment:
I think cfg test module should be at the end of the file

Reply:
resolved by https://github.com/near-cli-rs/interactive-clap/pull/26/commits/aad7bda7a14aeae49fe5a0407e879f8e08e0719b